background_scripts/main.js

```diff
@@ -271,10 +271,15 @@ const BackgroundCommands = {
     }
 
     if (request.registryEntry.options.incognito || request.registryEntry.options.window) {
+      // Firefox does not support about:newtab in chrome.windows.create.
+      const legalUrls = request.urls.filter((url) => url !== Settings.defaults.newTabUrl);
       const windowConfig = {
-        url: request.urls,
+        url: legalUrls,
         incognito: request.registryEntry.options.incognito || false
       };
+      if (windowConfig.url.length === 0) {
+        delete windowConfig.url;
```

Comment:
In my testing, this step was unnecessary -- if urls was an empty array, the new window still opened as expected.